### ISSUE TITLE
Introduce docGroupRef for defined VCs

### ISSUE THREAD

**Issue by Akron** [KorAP/Koral#49]
To make working with defined VCs (as implemented by @margaretha) more flexible, I propose an additional corpus/collection object type ```koral:docGroupRef```, that references a defined VC in the corpus/collection query.

```
{
  "@type" : "koral:docGroupRef",
  "ref" : "https://korap.ids-mannheim.de/@ndiewald/MyCorpus"
}
```

The reference is a unique ID that will be resolved by the KoralQuery consumer (in our case Kustvakt) - by injecting the requested KoralQuery fragment in the corpus definition (or by injecting a stored ID, in case the VC is persistant in the backend as a list of text IDs, see our discussion on dynamic vs. persistant VCs). Therefore defined VCs can also be part of more complex VCs.

In the frontend's VC builder the reference can be shown and created like "referTo @ndiewald/MyCorpus" along with a ```...```-symbol (in addition to ```x```, ```and``` and ```or```). When the user clicks on ```...``` the reference is resolved and shown in the frontend.

In the Koral VC language we would need to introduce a syntax like ```referTo {URL}```.

**Update**: Removed desc and type from the example, and replaced ```@ref``` with ```ref``` after comments by @margaretha.

**Comment by margaretha** [comment]
Implemented in 1ef86de1f19d4a6a4554879b31ed7c2a098f8fca.

**Comment by Akron** [comment]
Documented in https://github.com/KorAP/Koral/commit/b7bbb97d92d3becd33447b8433136d2e7a62a3d9 .
